.travis.yml

```diff
@@ -14,6 +14,10 @@ matrix:
             - libncurses5-dev
             - libpcre2-dev
             - python
+            - python3
+            - python3-pip
```

Comment:
Just make this `python3-pexpect`, and drop the `pip` lines on everything except macOS - the system packages install faster and are fine (or I can do it after merge).

Reply:
Awesome, I'll do it